Sources/CSG.swift

```diff
@@ -326,10 +326,10 @@ public extension Mesh {
             }
             let rect = Polygon(
                 unchecked: [
-                    Vertex(Vector(-radius, radius, 0), -normal, .zero),
-                    Vertex(Vector(radius, radius, 0), -normal, Vector(1, 0, 0)),
-                    Vertex(Vector(radius, -radius, 0), -normal, Vector(1, 1, 0)),
-                    Vertex(Vector(-radius, -radius, 0), -normal, Vector(0, 1, 0)),
+                    Vertex(Vector(-radius, radius, 0), -Direction(normal), .zero),
+                    Vertex(Vector(radius, radius, 0), -Direction(normal), Vector(1, 0, 0)),
```

Comment:
This is the only suspicious place I could find in the change set. @nicklockwood can you try what happens when `-Direction(normal)` is changed to `Direction(-normal)`?